src/RA_Achievement.h

```diff
@@ -33,8 +33,8 @@ class Achievement
     Achievement() noexcept;
 
 public:
-    void Clear();
-    bool Test();
+    void Clear() noexcept;
+    [[gsl::suppress(f.6)]] bool Test() noexcept;
```

Comment:
Consider using the `GSL_SUPPRESS()` macro. It adds some clang compatibility, and makes it easier to disable if we ever support a platform that doesn't understand it.